sspl_test/common.py

```diff
@@ -305,6 +311,39 @@ def send_enclosure_request(resource_type, resource_id):
     }
     write_to_egress_msgQ(request)
 
+
+def send_thread_controller_actuator_request(module_name, state):
+    request = {
+        "username":"sspl-ll",
+        "expires":3600,
+        "description":"Seagate Storage Platform Library - Actuator Request",
+        "title":"SSPL-LL Actuator Request",
+        "signature":"None",
+        "time":"2021-08-20 12:23.10.071170",
```

Comment:
why don't we use the actual time here? 😅

Reply:
Sure

Reply:
Done.